### DISCUSSION TITLE
No module named 'orjson'

### DISCUSSION THREAD

**Discussion by MaxHerzberg88** [DIGITALCRIMINALS/OnlyFans#492] [Q&A]
Cheers!
Can someone explain to me what I have to do in order to fix this issue?
![Screenshot 2022-06-12 205520](https://user-images.githubusercontent.com/98352297/173249139-3f9ade25-c1ca-4091-8d42-7863467affa6.png)

**Comment by m4rc1nt0sch** [comment]
Do you have Python3 installed? The Scraper uses Python and py.libraries, it won't work without.

**Reply by Tyemnata** [reply]
I have the same issue and I literally just installed python.

**Reply by kokomo1234** [reply]
Try to do the command 'pip install orjson' i was missing a bunch of things and just pip installed them all and it seems to work now
